.github/workflows/run-ci-for-external-forks.yml

```diff
@@ -32,13 +35,15 @@ jobs:
           team: wrangler
 
       - name: Stop workflow if user is not a team member
-        if: ${{ steps.teamAffiliation.outputs.isTeamMember == false }}
+        if: ${{ steps.teamAffiliation.outputs.isTeamMember == 'false' }}
         run: |
-          echo "You must be on the "wrangler" team to trigger this job."
+          echo "You must be on the \"wrangler\" team to trigger this job."
           exit 1
 
       - name: "Checkout PR"
-        run: gh pr checkout ${{ inputs.pr-number }} -b run-ci-on-behalf-of-${{ inputs.pr-number }} -f
+        run: |
+          gh pr checkout ${{ inputs.pr-number }} --branch run-ci-on-behalf-of-${{ inputs.pr-number }}
+          git reset --hard ${{ inputs.commit-sha }}
```

Comment:
Can we check what happens if this fails? What if we get the SHA wrong?
Is there a chance it will quietly fail and not reset the HEAD, leaving us at the potentially problematic HEAD commit?

Reply:
As long as the workflow is not configured somehow to try to keep going... But I think we are good.
We can always test it quite easily once we have landed this.

Reply:
@petebacondarwin I think we're all good 🙂👍